docs/c-lightning/overview.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: calling c-lightning rpc and building plugins.
-category: c-lightning support
+title: overview
+category: c-lightning
```

Comment:
@joemphilips I think CLightning project got renamed to LightningCore

Reply:
Actually its "core lightning" I believe this is to keep compatibility with old "c-lightning"
but yeah why not. I will rename it to "core-lightning".

Reply:
Actually I use the term "c-lightning" all over the place already.
I think I will keep it as is.